CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - PSR-14 event for site search tracking (#4)
+- PSR-14 event to enrich the script tag with additional attributes.
```

Comment:
Add the issue number (#5) and - if you like - your name, like "Thanks to ..."

Reply:
Added, but i don't need my name there ;-)